configs/centerpoint/argoverse_centerpoint_pp_02voxel_circle_nms.py

```diff
@@ -0,0 +1,192 @@
+    ),
+    backbone=dict(type="PointPillarsScatter", ds_factor=1, norm_cfg=norm_cfg,),
+    neck=dict(
+        type="RPN",
+        layer_nums=[3, 5, 5],
+        ds_layer_strides=[2, 2, 2],
+        ds_num_filters=[64, 128, 256],
+        us_layer_strides=[0.5, 1, 2],
+        us_num_filters=[128, 128, 128],
+        num_input_features=64,
+        norm_cfg=norm_cfg,
+        logger=logging.getLogger("RPN"),
+    ),
+    bbox_head=dict(
+        # type='RPNHead',
+        type="CenterHead",
+        mode="3d",
+        in_channels=sum([128, 128, 128]),
+        norm_cfg=norm_cfg,
+        tasks=tasks,
+        dataset='argoverse',
+        weight=0.25,
+        code_weights=[1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0],
+        common_heads={'reg': (2, 2), 'height': (1, 2), 'dim': (3, 2), 'rot': (2, 2)},  # (output_channel, num_conv)
+        encode_rad_error_by_sin=False,
+        direction_offset=0.0,
+        bn=True
+    ),
+)
+
+assigner = dict(
+    target_assigner=target_assigner,
+    out_size_factor=get_downsample_factor(model),
+    dense_reg=1,
+    gaussian_overlap=0.1,
+    max_objs=500,
+    min_radius=2,
+)
+
+
+train_cfg = dict(assigner=assigner)
+
+
+test_cfg = dict(
+    post_center_limit_range=[-61.2, -61.2, -10.0, 61.2, 61.2, 10.0],
+    max_per_img=500,
+    max_pool_nms=False,
+    min_radius=[4, 12, 10, 1, 0.85, 0.175],
+    post_max_size=83,
+    score_threshold=0.1,
+    pc_range=[-51.2, -51.2],
+    out_size_factor=get_downsample_factor(model),
+    voxel_size=[0.2, 0.2],
+    circle_nms=True
+)
+
+
+# dataset settings
+dataset_type = "ArgoverseDataset"
+data_root = "data/argoverse/sample"
+
+train_preprocessor = dict(
+    mode="train",
+    shuffle_points=True,
+    gt_loc_noise=[0.0, 0.0, 0.0],
+    gt_rot_noise=[0.0, 0.0],
+    global_rot_noise=[-0.3925, 0.3925],
+    global_scale_noise=[0.95, 1.05],
+    global_rot_per_obj_range=[0, 0],
+    global_trans_noise=[0.2, 0.2, 0.2],
+    remove_points_after_sample=False,
+    gt_drop_percentage=0.0,
+    gt_drop_max_keep_points=15,
+    remove_unknown_examples=False,
+    remove_environment=False,
+    db_sampler=None,
+    class_names=class_names,
+)
+
+val_preprocessor = dict(
+    mode="val",
+    shuffle_points=False,
+    remove_environment=False,
+    remove_unknown_examples=False,
+)
+
+voxel_generator = dict(
+    range=[-51.2, -51.2, -5.0, 51.2, 51.2, 3.0],
+    voxel_size=[0.2, 0.2, 8],
+    max_points_in_voxel=20,
+    max_voxel_num=30000,
+)
+
+train_pipeline = [
+    dict(type="Preprocess", cfg=train_preprocessor),
+    dict(type="Voxelization", cfg=voxel_generator),
+    dict(type="AssignLabel", cfg=train_cfg["assigner"]),
+    dict(type="Reformat"),
+]
+test_pipeline = [
+    dict(type="Preprocess", cfg=val_preprocessor),
+    dict(type="Voxelization", cfg=voxel_generator),
+    dict(type="AssignLabel", cfg=train_cfg["assigner"]),
+    dict(type="Reformat"),
+]
+
+train_anno = "data/nuScenes/infos_train_10sweeps_withvelo_filter_True.pkl"
```

Comment:
Are we not using the Argoverse data for this?

Reply:
These are not used. I'll remove them.